test/removed-dub-obj.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+. $(dirname "${BASH_SOURCE[0]}")/common.sh
+cd ${CURR_DIR}/removed-dub-obj
+rm -rf .dub
+
+${DUB} build --compiler=${DC}
+
+[ -d ".dub/obj" ] && die $LINENO '.dub/obj was found'
```

Comment:
could you check also for existence of the object file(s) in the new place?

Reply:
sure